src/neo/SmartContract/NefFile.cs

```diff
@@ -37,9 +38,9 @@ public class NefFile : ISerializable
         public Version Version { get; set; }
 
         /// <summary>
-        /// Script Hash
+        /// Abi
         /// </summary>
-        public UInt160 ScriptHash { get; set; }
+        public ContractAbi Abi { get; set; }
```

Comment:
I don't understand the value of storing just the ABI in the NEF file. You'll still need the non ABI parts of the manifest to deploy the contract right?

Reply:
never mind, saw the thread on the other issue